crates/astria-conductor/src/sequencer/client.rs

```diff
@@ -38,7 +39,10 @@ impl SequencerGrpcClient {
         let uri: Uri = sequencer_uri
             .parse()
             .wrap_err("failed parsing provided string as Uri")?;
-        let endpoint = Endpoint::from(uri.clone());
+        let mut endpoint = Endpoint::from(uri.clone());
+        if uri.scheme() == Some(&http::uri::Scheme::HTTPS) {
+            endpoint = endpoint.tls_config(ClientTlsConfig::new().with_enabled_roots())?;
```

Comment:
This is missing error context. Report that you failed to set the tls config using `wrap_err`